.github/workflows/build.yml

```diff
@@ -76,6 +76,12 @@ jobs:
           path: ./bin/neo-go*
           if-no-files-found: error
 
+      - name: Attach binary to the release as an asset
+        if: ${{ github.event_name == 'release' }}
+        run: gh release upload ${{ github.event.release.tag_name }} ./bin/neo-go-${{ matrix.os.bin-name }}-${{ matrix.arch }}${{ (matrix.os.bin-name == 'windows' && '.exe') || '' }}
```

Comment:
There's a batter way to replace these long tag name and file name with env vars, but it requires two different commands for Windows/others, thus I've desided to keep it as it is now.